packages/api/src/graphql/mutation.public.ts

```diff
@@ -79,6 +80,26 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
         const session = authenticateUser()
         return session ? await dbAdapter.session.deleteUserSessionByToken(session.token) : false
       }
+    },
+
+    // Comment
+    // =======
+    createComment: {
+      type: GraphQLNonNull(GraphQLComment),
+      args: {input: {type: GraphQLNonNull(GraphQLCommentInput)}},
```

Comment:
I suggest to use a completely new input object

Reply:
Can you check it now?